modules/ROOT/pages/policies-policy-level.adoc

```diff
@@ -0,0 +1,98 @@
+= Policy Level
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
```

Comment:
shouldn't we include resource level as keyword here?

Reply:
Good point! Thanks! Done!